src/app/shared/components/filters-list/city-filter/city-filter.component.ts

```diff
@@ -113,13 +130,8 @@ export class CityFilterComponent implements OnInit, OnDestroy {
 
   changeCity(): void {
     this.isDispalyed = false;
-    if (this.settlement != Constants.KYIV) {
-      this.geolocationService.confirmCity(Constants.KYIV, false);
-      this.actions$.pipe(ofActionCompleted(GetCodeficatorSearch)).subscribe(() => this.setInputFocus());
-      return;
-    }
-
-    this.setInputFocus();
+    this.store.dispatch(new CleanCity());
+    this.actions$.pipe(ofActionCompleted(GetCodeficatorSearch), takeUntil(this.destroy$)).subscribe(() => this.setInputFocus());
```

Comment:
on out of focus, populate previously selected city (that is stored in the state)